src/integration/blockchain/shared/services/blockchain-registry.service.ts

```diff
@@ -22,9 +24,10 @@ export class BlockchainRegistryService {
     private readonly polygonService: PolygonService,
     private readonly baseService: BaseService,
     private readonly moneroService: MoneroService,
+    private readonly btcService: BtcService,
   ) {}
 
-  getClient(blockchain: Blockchain): EvmClient | MoneroClient {
+  getClient(blockchain: Blockchain): EvmClient | MoneroClient | BtcClient {
     return this.getService(blockchain).getDefaultClient();
   }
```

Comment:
This will throw an error for bitcoin, as the type is not defined. Maybe we should by default return the Input client?